packages/richtext-lexical/src/features/indent/client/index.tsx

```diff
@@ -81,10 +86,49 @@ const toolbarGroups: ToolbarGroup[] = [
   ]),
 ]
 
+const IndentPlugin: PluginComponent<IndentFeatureProps> = (props) => {
+  const [editor] = useLexicalComposerContext()
+  const { disabledNodes } = props.clientProps
+
+  useEffect(() => {
+    if (!editor) {
```

Comment:
Change to

```ts
 if(!editor || !disabledNodes?.length) {
   return
 }
```

to avoid calling `$handleIndentAndOutdent` and registering those commands below.

That way you should be able to remove the `if (!disabledNodes` condition below